data/strings/strings.txt

```diff
@@ -29869,3 +29869,94 @@
     vi = Bản đồ
     zh-Hans = 地图
     zh-Hant = 地圖
+
+  [drive_through_yes]
+    en = Drive-through
+    af = Deur-ry
+    ar = الشراء من السيارة
+    az = Sürüşmə
+    be = Праехаць
+    bg = Карам през
+    ca = Condueix per
+    cs = Průjezd
+    da = Køre gennem
+    de = Durchfahrtsbedienung
```

Comment:
`de = Drive-Through`
There is no meaningful german translation I'd be familiar with.

Reply:
You're probably correct in that 'Drive-In' is wider known, but it's badly translated which I'd speculate we can blame McDonald's marketing for.
Like "_Let's not use a name se germans can't pronounce!_".
I wouldn't be surprised if 'Drive-In' is 99% used by McDonald's allone (or even trademarked), but they're just everywhere so everybody knows the term.

[https://wiki.openstreetmap.org/wiki/DE:Key:drive_through](DE:Key:drive_through)
> Bitte beachte, dass sich im deutschsprachigen Raum für diese Einrichtungen fälschlicherweise der Begriff "Drive-in" durchgesetzt hat. Hierbei wird man jedoch nicht beim Durchfahren bedient, sondern nachdem man sein Fahrzeug geparkt hat (z.B. beim Autokino).

If my Mäckes-bashing is wrong though, I'd still go for 'Drive-Through' as the (english) language barrier did move quite a bit since whenever 'Drive-Ins' became a thing in germany.